### PR TITLE
[teleop] Added gate location on auton gui map

### PR DESCRIPTION
**Pull request by catkims**
Displays gate location on the map.

### CODE REVIEW

File: pip_deps/jetson_requirements.txt [hunk 1]
```diff
@@ -1,6 +0,0 @@
-numpy==1.16.2
-<<<<<<< HEAD
-=======
-scipy==1.5.4
->>>>>>> main
-odrive==0.5.1
```
Comment (CameronTressler) [inline pip_deps/jetson_requirements.txt:6]: @cgiger00 @zwgold what should be in this file?

Comment (zwgold) [reply pip_deps/jetson_requirements.txt:6]: I mentioned this in the simulator path points PR, but 99% sure scipy should no longer be in there. @cgiger00 plz confirm

Comment (cgiger00) [reply pip_deps/jetson_requirements.txt:6]: yep

**Review by cgiger00 — APPROVED** [review]
to nitpick maybe the gates could be a different icon?

**Review by CameronTressler — APPROVED** [review]

**Pull request merged**